app/views/notes/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
```

Comment:
`<%= notice %>` is already shown in `app/views/layouts/application.html.erb`, right? Wouldn't that show twice then?

Reply:
Yep. I removed it in the app/view/notes/show.html.erb too for the same reason